src/lib.rs

```diff
@@ -2044,6 +2044,8 @@ impl Build {
                     } else {
                         format!("{}.exe", gnu)
                     }
+                } else if target.contains("ios") {
```

Comment:
Could this perhaps use `-ios-` as a search string to be a bit more flexible? The string "ios" is a little short unfortunately...

Reply:
iOS has 7 different targets. See [platform-support](https://doc.rust-lang.org/beta/rustc/platform-support.html). 

- aarch64-apple-ios
- x86_64-apple-ios
- aarch64-apple-ios-macabi
- armv7-apple-ios
- armv7s-apple-ios
- i386-apple-ios
- x86_64-apple-ios-macabi

It looks like `-apple-ios` is the better choice.

Reply:
Sounds good to me!

Reply:
Done